src/typeahead/typeahead.directive.ts

```diff
@@ -402,7 +408,7 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
               );
           })
         )
-        .subscribe((matches: TypeaheadMatch[]) => {
+        .subscribe((matches: any[]) => {
```

Comment:
according to chat with @IraErshova , there will be separate PR for fix of types in typeahead, because, this TypeaheadMatch type here was not exactly correct

Reply:
Thanks for your comment!

Reply:
[PR for typing](https://github.com/valor-software/ngx-bootstrap/pull/5647/commits/e21f68c18ce5d522f55ad13f798bfa08b3baca8d)